src/VirtualClient/VirtualClient.Actions/SPECview/SpecViewExecutor.cs

```diff
@@ -31,6 +29,17 @@ public class SpecViewExecutor : VirtualClientComponent
     {
         private const string VisualStudioCRuntimePackageName = "visualstudiocruntime";
         private const string RenamePrefix = "hist_";
+        private readonly IDictionary<string, string> viewsetLogFileNameMapping = new Dictionary<string, string>()
+        {
+            { "3dsmax", "3dsmax-07" },
+            { "catia", "catia-06" },
+            { "creo", "creo-03" },
+            { "energy", "energy-03" },
+            { "maya", "maya-06" },
+            { "medical", "medical-03" },
+            { "snx", "snx-04" },
+            { "sw", "solidworks-07" }
```

Comment:
is there a reason sw is the only exception here with acronyms?

Reply:
The SPECviewperf software dictates these naming. I want to ask them same question as well :-(.